serde-api/src/main/java/io/micronaut/serde/Deserializer.java

```diff
@@ -128,7 +128,7 @@ default ConversionService getConversionService() {
          * @return {@code true} iff any of the given views is enabled.
          */
         default boolean hasView(Class<?>... views) {
-            return false;
+            return true;
```

Comment:
why was this default changed to `true`?

Reply:
Jackson databind behavior is that if no view is specified, all fields are included.